src/Reducers/moviesReducer.ts

```diff
@@ -0,0 +1,52 @@
+import * as types from '../Constants/ActionTypes';
+
+export default function moviesReducer(
```

Comment:
In complex scenarios try to use `combineReducers`

Reply:
sure, once I see I need to use multiple reducers I can try with the same suggestion.